embedding/csharp/Tizen.Flutter.Embedding/Interop/Interop.cs

```diff
@@ -24,6 +30,7 @@ public struct FlutterDesktopWindowProperties
             public bool focusable;
             [MarshalAs(UnmanagedType.U1)]
             public bool top_level;
+            public int renderer_type;
```

Comment:
Casting should not be necessary.
```suggestion
            public FlutterDesktopRendererType renderer_type;
```

Reply:
done